runtime/src/chpl-privatization.c

```diff
@@ -87,3 +98,10 @@ int64_t chpl_numPrivatizedClasses(void) {
   chpl_sync_unlock(&privatizationSync);
   return ret;
 }
+ 
+// Clean up all resources for memory checkers.
+void chpl_privatization_exit(void) {
+  // Destroy last instance
+  chpl_mem_free(chpl_privateObjects, 0, 0);
+  chpl_privateObjects = NULL;
+}
```

Comment:
I think we expect a newline at the end of the file and there isn't one, if I'm interpreting the red mark above correctly.

Reply:
Done.